pkgs/development/libraries/openbabel/default.nix

```diff
@@ -1,34 +1,63 @@
-{stdenv, fetchurl, fetchpatch, cmake, zlib, libxml2, eigen, python, cairo, pcre, pkgconfig }:
+{stdenv, fetchurl, fetchpatch, cmake, zlib, libxml2, eigen, python, cairo, pcre, pkg-config, swig, rapidjson }:
```

Comment:
```suggestion
{stdenv, fetchurl, cmake, zlib, libxml2, eigen, python, cairo, pcre, pkg-config, swig, rapidjson }:
```

Reply:
ah good catch. The old version had a patch hosted on github which doesn't seem relevant anymore